lib/jobFactory.js

```diff
@@ -47,7 +57,21 @@ class JobFactory extends BaseFactory {
         c.state = 'ENABLED';
         c.archived = false;
 
-        return super.create(c);
+        return super.create(c).then((job) => {
+            if (!c.permutations ||
+                !c.permutations[0].annotations ||
+                !c.permutations[0].annotations['beta.screwdriver.cd/buildPeriodically']) {
```

Comment:
That doesn't work on array, since `permutations` is an array. 
I can try to simplify it...

Reply:
Ah, I remember why I didn't do this in the first place. Even if assuming `permutations[0]` is there, the `hoek.reach` will try to do a reach for `cd` inside `screwdriver` inside `beta` instead of interpreting the whole thing `beta.screwdriver.cd` as a `string`.